stories/0-Examples.stories.js

```diff
@@ -1,47 +1,21 @@
 import React from 'react';
```

Comment:
I thought I had -- I think I messed up something in the merge because another file came back too (that was messing up the Travis tests when I first pushed this) -- will remove it